datafusion/core/src/datasource/physical_plan/mod.rs

```diff
@@ -97,6 +118,106 @@ impl FileSinkConfig {
     pub fn output_schema(&self) -> &SchemaRef {
         &self.output_schema
     }
+
+    /// Get object store from task context
+    pub fn get_object_store(
```

Comment:
Do we need this to be a part of this object's API? Seems like it is a one-liner and it may be prudent to avoid adding dependencies to this module just for this. Let's discuss.